metal/datasource_metal_project_ssh_key_test.go

```diff
@@ -76,7 +76,9 @@ func TestAccMetalProjectSSHKeyDataSource_ByID(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						datasourceName, "public_key", publicKeyMaterial),
 				),
-				ExpectNonEmptyPlan: true,
+				// Why was follwing flag set? The plan is applied and then it's empty.
```

Comment:
This may have been left over from experiments before finding https://github.com/packethost/terraform-provider-packet/pull/250#discussion_r490700522

Reply:
Removing the ExpectNonEmptyPlan and the comment seems good to me. We might want a comment explanation for the empty step that follows (maybe just a link to the terraform-provider-packet PR250 comment, or the hashicorp PR).